### PR TITLE
Example circuits 2

### PR DESCRIPTION
**Pull request by Keelando**
Enable example circuits from public/assets/demo_circuits folder

### CODE REVIEW

File: src/components/SelectExampleCircuit.tsx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    ApiHttpError,
+    CompilationResultSuccess,
+    NoServerResponse,
+    CompilerError,
+} from "../apiResponses"
+import queryString from "query-string"
+import Loader from "./Loader"
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const SelectExampleCircuit = ({ appState, setAppState }: AppStateProps) => {
```
Comment (alexnguyenn) [inline src/components/SelectExampleCircuit.tsx:19]:
```suggestion
const SelectExampleCircuit = ({ appState, setAppState }: AppStateProps): JSX.Element => {
```
To fix build error

File: src/components/SelectExampleCircuit.tsx [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+                            compilationIsLoading: false,
+                        })
+                        setExampleIsLoading(false)
+                    }
+                }
+
+                // Catch Axios Errors: API errors -> timeout, error compiling, or no response
+            })
+            .catch((error) => {
+                // needs more testing. First two requests after a long lambda cooldown period result in CORS / Network Errors
```
Comment (gwwatkin) [inline src/components/SelectExampleCircuit.tsx:102]: Can't replicate that error, to me it worked on the first try :thinking:

Comment (gwwatkin) [reply src/components/SelectExampleCircuit.tsx:102]: CORS is now handled by CloudFront's preset

**Review by gwwatkin — APPROVED** [review]
Some subtle changes here, great to see everything already working

**Pull request merged**